pandamium_datapack/data/pandamium/functions/startup.mcfunction

```diff
@@ -54,6 +54,7 @@ scoreboard objectives add spawnpoint_x dummy
 scoreboard objectives add spawnpoint_y dummy
 scoreboard objectives add spawnpoint_z dummy
 scoreboard objectives add spawnpoint_dim dummy
+scoreboard objectives add active_particle dummy
```

Comment:
Please change it to `active_particles` to have it the same as `particles`, also it makes more sense imo since you always have lots of particles, not just a single one